.github/workflows/ci-master.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v3
         with:
-          files: packages/cli/coverage/cobertura-coverage.xml,packages/nodes-base/coverage/cobertura-coverage.xml,packages/core/coverage/cobertura-coverage.xml,packages/workflow/coverage/cobertura-coverage.xml
+          files: packages/cli/coverage/cobertura-coverage.xml,packages/core/coverage/cobertura-coverage.xml,packages/design-system/coverage/cobertura-coverage.xml,packages/editor-ui/coverage/cobertura-coverage.xml,packages/nodes-base/coverage/cobertura-coverage.xml,packages/workflow/coverage/cobertura-coverage.xml
```

Comment:
Couldn't we use glob pattern matching here `packages/*/coverage/cobertura-coverage.xml`? It seems to be supported by the action: https://github.com/codecov/codecov-action/issues/657

Reply:
the issue was incorrectly closed. there is [currently no support for glob patterns](https://github.com/codecov/codecov-action/blob/main/src/buildExec.ts#L104).

Reply:
Nevermind, seems like they removed the support for v3 🤯 
https://github.com/codecov/codecov-action/issues/770